ramlfications/parser/main.py

```diff
@@ -73,6 +74,13 @@ def schemas():
             schemas.append({list(iterkeys(schema))[0]: value})
         return schemas or None
 
+    def types():
+        _types = _get(raml, "types")
+        if not _types:
+            return None
+        return dict([(k, create_type(k, v))
```

Comment:
I would like to preserve order here - to mimic what is in a RAML file - so perhaps create/return an OrderedDict.